.github/workflows/build-image.yml

```diff
@@ -0,0 +1,57 @@
+name: Publish Docker image
+on:
+  push:
+    tags:
+    - v[0-9]*
```

Comment:
Este formato solo permitiria tags de versiones major, como haria si quiero incluir un patch minor?.
Para eso necesitaria una ER asi no?

- v[0-9]+.[0-9]*

- v[0-9]+.[0-9]+.[0.9]*

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet te ayudará a escribir lo que necesiten.